_data/simple-icons.json

```
@@ -15100,6 +15100,16 @@
             "hex": "FFA633",
             "source": "https://foursquare.com/about/logos"
         },
+        {
+            "title": "Sway",
+            "hex": "68751C",
+            "source": "https://github.com/swaywm/sway/blob/master/assets/Sway_Tree.svg",
```

```suggestion
            "source": "https://github.com/swaywm/sway/blob/77b9ddabe2a97c5d04c30929b0f8cbde3470fdd7/assets/Sway_Tree.svg",
```
Please change to permalink when linking from GitHub